packages/upload-api/src/customer/get.js

```diff
@@ -24,7 +24,11 @@ const get = async ({ capability }, context) => {
     return { error: new UnknownProvider(capability.with) }
   }
 
-  const customer = await context.models.accounts.get(capability.nb.customer)
+  // TODO: we used to query the accounts table for this, but that only
+  // returns the DID we already have. we may want to query other tables in the future
+  // but for now I think we can skip this and drop the accounts table entirely
+  // const customer = await context.models.accounts.get(capability.nb.customer)
```

Comment:
got it! I've updated this PR along these lines - I introduced a new `getCustomer` function on `ProvisionsStorage` - since users essentially become "customers" when we provision storage for them this seems reasonable to me, but per my comment above we may want to introduce new interfaces for this soon...